website/project/model.py

```diff
@@ -506,9 +506,16 @@ def validate_title(value):
     if value is None or not value.strip():
         raise ValidationValueError('Title cannot be blank.')
 
+    value = sanitize.strip_html(value)
+
+    if value is None or not value.strip():
+        raise ValidationValueError('Invalid title.')
+
     if len(value) > 200:
         raise ValidationValueError('Title cannot exceed 200 characters.')
-    return True
+
+    return value
```

Comment:
The validate_title function does too many separate things, even though we have reusable components for each separate type of validation. ModularODM fields can support a *list* of validators separately, and I think that's a better course going forward (to reduce redundancy in our validation code).

Suggestions:
1. Change https://github.com/abought/osf.io/blob/develop/website/project/model.py#L608 from:
`title = fields.StringField(validate=validate_title)`

to:

```
from modularodm.validators import MaxLengthValidator
from framework.mongo.validators import string_required

title = fields.StringField(validate=[string_required, MaxLengthValidator(200)])
```

2. Make the post-processing (HTML removal) separate from validation.